### ISSUE TITLE
Error creating user.

### ISSUE THREAD

**Issue by karam94** [javebratt/ionic-firebase-auth#2]
Currently getting an "Error creating user" dialog when trying to register a user.
Any ideas? Am I missing any configuration in my firebase?

Thanks!

**Comment by javebratt** [comment]
Hey @karam94 what error are you getting? Are you using your own firebase url? Did you allow creation of users in your dashboard? The more specific you get the better.

**Comment by karam94** [comment]
Hey @javebratt.
As soon as I enter valid account details and click "Signup", a dialog appears in my browser (Google Chrome) and says:
"localhost:8100 says: Error creating user;"

I am using my own firebase URL and I have allowed the creation of users in my dashboard.
I only very recently created my firebase account, so I think it may be due to this ionic 1 auth technique being incompatible with the latest version of firebase perhaps? I'm not entirely sure.
The sign-up method in firebase that I've enabled is the Email/Password option.

Can't really think of anything else I can say to be more specific, unless you think providing you with my firebase URL would help?

Thanks!
